android/build.gradle

```diff
@@ -58,9 +58,10 @@ android {
     }
 }
 
+def target = new File(rootProject.projectDir.parent)
 // Include Rust crates in the build process
-apply from: "../connectors/cargokit/gradle/plugin.gradle"
+apply from: "../cargokit/gradle/plugin.gradle"
 cargokit {
-    manifestDir = "${rootProject.projectDir}/../native/hub"
+    manifestDir = "../${target.name}/native/hub"
     libname = "hub"
 }
```

Comment:
For this one, I'll test it with an actual Flutter project that I'm working on, which is inside totally different directory :)

Reply:
A sad story that failed when it was integrated into a specific flutter project and compiled to android on macOS platform.

Reply:
Yeah, I believe that's why we need to leave a PR at cargokit to use it as a submodule.... 👀

Reply:
Currently that gradle config doesn't seem to accept absolute paths

Reply:
> Currently that gradle config doesn't seem to accept absolute paths

Yes, that's true.

> Yeah, I believe that's why we need to leave a PR at cargokit to use it as a submodule.... 👀

At first, I was just worried that modifying the source code of cargokit was a complicated task, and it will add a lot of maintenance work to some projects that already rely on cargokit.

But when @knopp reply and mentioned the document today, maybe we could make cargokit better together.

Reply:
Yeah, I agree. His response was kind and fast when I left a PR there before.